pkg/cmd/activity.go

```diff
@@ -44,9 +45,10 @@ var activityCmd = &cobra.Command{
 }
 
 var activityGetCmd = &cobra.Command{
-	Use:   "get [log_id]",
-	Short: "Get workplace activity log",
-	Args:  cobra.MaximumNArgs(1),
+	Use:               "get [log_id]",
+	Short:             "Get workplace activity log",
+	Args:              cobra.MaximumNArgs(1),
+	ValidArgsFunction: activityLogIDsValidArgs,
```

Comment:
Does `ValidArgsFunction` do validation as well? I'm mainly concerned about a situation where `controllers.GetActivityLogsIDs` returns the first N results and we're trying to provide an ID for log N+1.

Reply:
`ValidArgsFunction` is purely additive, so it will only provide auto-completion when it can. It won't prevent the user from using an arg it doesn't know about.

Reply:
Awesome 👍